index.html

```diff
@@ -72,10 +72,10 @@
             </li>
             <li class="nav-item dropdown">
               <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownBlog" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                Other Pages
+                More Good Stuff Here
               </a>
               <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownBlog">
-                <a class="dropdown-item" href="full-width.html">Full Width Page</a>
+                <a class="dropdown-item" href="legal.html">Accessability</a>
```

Comment:
s/b: Accessibility
```suggestion
                <a class="dropdown-item" href="legal.html">Accessibility</a>
```

Reply:
Are we creating a new legal.html page or sticking with index.html?